supporting_content/attributes/null_handling.md

```diff
@@ -0,0 +1,29 @@
+# Null handling
+
+## Example usage scenarios
+
+Current values observed in billing data for various scenarios:
+
+| Provider  | Data set                 | Scenario                                                                         |
+|-----------|--------------------------|----------------------------------------------------------------------------------|
+| AWS       | CUR                      | Product_operation shows the type of operation used by a consumer. <br>However, in scenarios where the line item represents tax, this column is set to null. |
+| GCP       | Big Query Billing Export | Credits.name is null when a credit is not applied                                |
+| Microsoft | Cost Details             | `TODO` need example                                                              |
```

Comment:
ResourceGroup is null when a charge is not from a resource deployed to a resource group.